observations/marine/icec_amsr2_north.yaml.j2

```diff
@@ -45,6 +46,12 @@
 #    select_median: true
 #    action:
 #      name: reduce obs space
+  - filter: Domain Check
```

Comment:
We can't have amsr2 a passive, it's needed for the retrospective experiments.

Reply:
@guillaumevernieres Okay, I will fix that.